src/unitxt/operator.py

```diff
@@ -170,7 +173,7 @@ def _process_multi_stream(self, multi_stream: MultiStream) -> MultiStream:
     def _process_single_stream(self, stream: Stream, stream_name: str = None) -> Stream:
         return Stream(self._process_stream, gen_kwargs={"stream": stream, "stream_name": stream_name})
 
-    def is_should_be_processed(self, stream_name):
+    def _is_should_be_processed(self, stream_name):
         if (
             self.apply_to_streams is not None
```

Comment:
perhaps add tests for checking the functionality of filtering streams? (i.e. of `self.apply_to_streams`)